calculating_n50_assemblies.py

```diff
@@ -10,69 +10,80 @@ def main(in_dir = None, out_dir = None):
         sys.exit("Input directory or path incorrect. Please check")
 
     if os.path.exists(out_dir) == False:
-        os.makedirs(out_dir)
+        os.mkdir(out_dir)
     
-    file1 = open(os.path.join(out_dir+'/summary_concise.txt'),'w')
-    file2 = open(os.path.join(out_dir+'/summary_verbose.txt'),'w')
+    file1 = open(os.path.join(out_dir,'summary_concise.txt'),'w')
+    file2 = open(os.path.join(out_dir,'summary_verbose.txt'),'w')
 
+    fasta_ext = ['fasta','fna','ffn','faa','frn']
     #intializing assembly count
     assembly_count = 0
 
     #iterating through each assembly file
-    for file in glob.glob(os.path.join(in_dir+'/*.gz')):
-        file2.write ("Opened file {}\n".format(file))
-        assembly_count += 1
-        with gzip.open(file, mode = 'rt') as f:
+    for file in glob.glob(os.path.join(in_dir,'*.gz')):
+        array_file = file.split('.')
+        approval = 0
+        for w in fasta_ext:
```

Comment:
Also instead of looping through the array, you would be better of suited python array method to check if a value is present in an array.

Reply:
> os.path contains utilities to get the extension of a file, please refer to the documentation. If after searching you cannot find the method I'm talking about please ask me.

Ok I see why you structured it that way as you only allow .gz files so [-2] is fine.

Reply:
> Also instead of looping through the array, you would be better of suited python array method to check if a value is present in an array.

Something like "if array_file[-2] in fasta_ext:" ?

Reply:
> > Also instead of looping through the array, you would be better of suited python array method to check if a value is present in an array.
> 
> Something like "if array_file[-2] in fasta_ext:" ?

Did you try it? Does it work?

Reply:
Yes

Reply:
Ok, perfect